libutils/sequence.c

```diff
@@ -407,3 +407,22 @@ int SeqStringLength(Seq *seq)
 
     return total_length;
 }
+
+void SeqRemoveNulls(Seq *s)
+{
+    int length = SeqLength(s);
+    int from = 0;
+    int to = 0;
+    while (from < length)
+    {
+        if (s->data[from] == NULL)
+        {
+            ++from;
+            continue;
+        }
+        s->data[to] = s->data[from];
+        ++from;
+        ++to;
```

Comment:
I'd prefer ``else`` here instead of ``continue``, but YMMV.

Reply:
I (obviously) preferred the `continue`, but maybe I'll change it. Just because it's you ;)